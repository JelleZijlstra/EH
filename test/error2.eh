#!/usr/bin/ehi
# Test error handling
$echo 3
$echo +
$echo 2
