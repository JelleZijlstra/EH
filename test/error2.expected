3
syntax error, unexpected T_SEPARATOR: EH parsing error
2
